### ISSUE TITLE
Incorrect handling of arrays?

### ISSUE THREAD

**Issue by Anahkiasen** [sindresorhus/query-string#66]
Currently if `query-string` encounters an array it encodes it without brackets:

``` js
> require('query-string').stringify({foo: ['bar', 'baz']})
'foo=bar&foo=baz'
```

However that seems to go against the default browser behavior or one of other languages such as PHP. Shouldn't it be `foo[]=bar&foo[]=baz`?

I know this is kind of a grey area of the RFC but that struck me as odd, is this intended?

**Comment by SamVerschueren** [comment]
I believe this is intended. All node frameworks handle it correctly.

**Comment by abdulrahmancz** [comment]
What about adding config option for stringify and parse methods?

**Comment by johannesnagl** [comment]
The Gozala/querystring repo has the same PR: https://github.com/Gozala/querystring/pull/10

**Comment by sindresorhus** [comment]
Yeah sure, if someone does a good pull request with tests.

**Comment by rapzo** [comment]
See #79

**Comment by sindresorhus** [comment]
Fixed by #79.
